lib/mongoose_push_web/plugs/cast_and_validate.ex

```diff
@@ -0,0 +1,133 @@
+defmodule MongoosePushWeb.Plug.CastAndValidate do
+  @behaviour Plug
+
+  @impl Plug
+  def init(opts) do
+    OpenApiSpex.Plug.CastAndValidate.init(opts)
```

Comment:
How about aliasing `OpenApiSpex.Plug.CastAndValidate` to `OpenApiSpexValidator` or something similar?

Reply:
I personally don't think it will improve the readability, as currently I strictly differentiate my wrapping `MongoosePushWeb.Plug.CastAndValidate` from the original plug `OpenApiSpex.Plug.CastAndValidate`.